Source/WebKit/Shared/WebPageCreationParameters.h

```diff
@@ -187,7 +187,7 @@ struct WebPageCreationParameters {
 #if USE(WPE_RENDERER)
     IPC::Attachment hostFileDescriptor;
 #endif
-#if PLATFORM(WIN)
+#if USE(GRAPHICS_LAYER_TEXTURE_MAPPER)
     uint64_t nativeWindowHandle;
```

Comment:
nativeWindowHandle is not only for GraphicsLayerTextureMapper, but also for GraphicsLayerWC. And, I have a plan to remove GraphicsLayerTextureMapper after removing WinCairo WK1 and migrating Microsoft Playwright to GraphicsLayerWC.
I think it should be `#if USE(GRAPHICS_LAYER_TEXTURE_MAPPER) || USE(GRAPHICS_LAYER_WC)`.

Reply:
Ok updated @fujii